running/ReplacingDocker/security_demo.sh

```diff
@@ -0,0 +1,461 @@
+#!/bin/sh
+
+# devconf-demos.sh demo script.
+# This script will demonstrate security features of buildah,podman,skopeo and cri-o
+
+# Setting up some colors for helping read the demo output.
+# Comment out any of the below to turn off that color.
+bold=$(tput bold)
+blue=$(tput setaf 4)
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${blue}$1${reset}"
+}
+
+# Initial setup
+setup() {
+    rpm -q podman buildah audit >/dev/null
+    if [[ $? != 0 ]]; then
+	echo $0 requires the podman, buildah and audit packages be installed
+	exit 1
+    fi
+    command -v docker > /dev/null
+    if [[ $? != 0 ]]; then
+	echo $0 requires the docker package be installed
+	exit 1
+    fi
+    sudo cp /usr/share/doc/audit/rules/10-base-config.rules /etc/audit/rules.d/audit.rules
+    sudo augenrules --load > /dev/null
+    sudo systemctl restart auditd 2> /dev/null
+    sudo systemctl restart docker
+    sudo podman kill -a
+    sudo podman rm -af
+    podman kill -a
+    sleep 1
+    podman rm -af
+    sudo podman rmi $(sudo podman images | grep none | awk '{print $3}')
+    sudo docker pull ubuntu
+    clear
+}
+
+buildah_image() {
+    sudo podman images  | grep -q -w buildah-ctr
+    if [[ $? != 0 ]]; then
+	sudo podman build -t buildah-ctr -f Dockerfile.buildah .
+    fi
+}
+
+intro() {
+    read_color "DevConf Demos!  Buildah, Podman, Skopeo, CRI-O Security"
+    echo ""
+    clear
+}
+
+buildah_minimal_image() {
+  # Buildah from scratch - minimal images
+  read_color "Buildah from scratch - building minimal images"
+  echo ""
+
+  read_color "--> ctr=\$(sudo buildah from scratch)"
+  ctr=$(sudo buildah from scratch)
+  echo_color $ctr
+  echo ""
```

Comment:
Your call, but I think it works best if you use 'read_color' as you have it here, but then using the 'echo_color' for each  line that contains an actual command.  So that way  the command used is in bold, the results in default.  I think it makes it easier to track by eye.

Reply:
looking.. trying it out, thanks